english/khudi/buland_very_high.md

```diff
@@ -0,0 +1,11 @@
+Make your self-confidence and capabilities so high,
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.